src/axom/bump/extraction/TableBasedExtractor.hpp

```diff
@@ -1170,6 +1170,8 @@ class TableBasedExtractor
     axom::for_all<ExecSpace>(
       selectedZonesView.size(),
       AXOM_LAMBDA(axom::IndexType szIndex) {
+        // Avoid first-capture in constexpr-if context error
+        (void)selection;
```

Comment:
Capture these variables outside of "if constexpr".

Documentation about this:
https://docs.nvidia.com/cuda/cuda-programming-guide/05-appendices/cpp-language-support.html#extended-lambda-restrictions
 
The exact rule line in question: "A variable cannot be implicitly captured inside an if-constexpr block that is lexically nested inside an extended lambda unless the variable has been implicitly captured outside the if-constexpr block or appears in the extended lambda’s explicit capture list."